src/pages/home/report/ReportActionCompose.js

```diff
@@ -241,24 +245,61 @@ class ReportActionCompose extends React.Component {
         return this.props.translate('reportActionCompose.writeSomething');
     }
 
+    /**
+     * Set text input value and selection, then call updateComment with newText parameter
+     *
+     * @param {String} newText
+     * @param {Object} selection: {start: number, end: number}
+     */
+    setTextAndSelectionOfTextInput(newText, selection) {
+        if (Platform.OS === 'web') {
```

Comment:
We don't use this. [Check this](https://github.com/Expensify/App#platform-specific-file-extensions).

Reply:
Alright, I will try other way.